var/spack/repos/builtin/packages/mysql/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Mysql(CMakePackage):
+    homepage = "http://dev.mysql.com"
+    url      = "https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.11.tar.gz"
+
+    version('8.0.11', '38d5a5c1a1eeed1129fec3a999aa5efd')
+    version('5.7.22', '269935a8b72dcba2c774d8d63a8bd1dd')
+
+    depends_on('boost@1.66.0', when='@8.0.11')
```

Comment:
does it only work with these specific boost versions?

Reply:
Yes, each version of `mysql` requires a specific version of `boost` at build time. See `BOOST_PACKAGE_NAME` in `cmake/boost.cmake` in the downloaded tarball.

> The Boost C++ libraries are required to build MySQL (but not to use it). MySQL compilation requires a particular Boost version. Typically, that is the current Boost version, but if a specific MySQL source distribution requires a different version, the configuration process will stop with a message indicating the Boost version that it requires.

https://dev.mysql.com/doc/refman/8.0/en/source-installation.html